types/params.go

```diff
@@ -83,6 +88,23 @@ func (a ABCIParams) VoteExtensionsEnabled(h int64) bool {
 	return a.VoteExtensionsEnableHeight <= h
 }
 
+// FeatureParams configure parameters of different features of CometBFT.
+type FeatureParams struct {
+	VoteExtensionsEnableHeight *int64 `json:"vote_extensions_enable_height"`
```

Comment:
Shouldn't it be a `types.Int64Value`?

Reply:
Moreover, shouldn't we add the ABCI version after?

Reply:
We use the Int64Value only to make protobuffer entries nullable. The domain types are kept "simpler".